.github/workflows/daily_test.yml

```diff
@@ -0,0 +1,81 @@
+name: daily_flow_test
+on:
+  push
```

Comment:
with this, will it run daily or just on updates to the repo / pushes?

Reply:
The issue in question specifies it should be run daily, but for now it runs on pushes just so I have feedback on my test runs.